core/src/main/java/com/linecorp/armeria/server/encoding/HttpEncoders.java

```diff
@@ -110,7 +113,7 @@ private static HttpEncodingType determineEncoding(String acceptEncoding) {
 
         if (!encodings.isEmpty()) {
             final Entry<HttpEncodingType, Float> entry = Collections.max(encodings.entrySet(),
-                                                                         Entry.comparingByValue());
+                    Entry.comparingByValue());
```

Comment:
Can you check if you have set up your IDE correctly for code style and revert all changes caused by this?
https://armeria.dev/community/developer-guide#setting-up-your-ide

Reply:
Thanks for the review. I did setup the code style for Eclipse. But I think I'm using a different version of Eclipse which didnt have all the options provided in the screenshot in the documentation. I'll fix that.